Subject: [PATCH] CS461x gameport code isn't being included in build
Date:	Tue, 04 Jan 2005 04:14:02 +0000
X-BK-ChangeSetKey: rjmx@rjmx.net[torvalds]|ChangeSet|20050104041402|59415
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2136.3.10, 2005/01/03 20:14:02-08:00, rjmx@rjmx.net

	[PATCH] CS461x gameport code isn't being included in build
	
	With Cal Peake <cp@absolutedigital.net>
	
	I've found a typo in drivers/input/gameport/Makefile in kernel 2.6.9 which
	effectively prevents the CS461x gameport code from being included.
	
	Signed-off-by: Ron Murray <rjmx@rjmx.net>
	Signed-off-by: Cal Peake <cp@absolutedigital.net>
	Signed-off-by: Andrew Morton <akpm@osdl.org>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 Makefile |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/input/gameport/Makefile b/drivers/input/gameport/Makefile
--- a/drivers/input/gameport/Makefile	2005-01-03 22:10:04 -08:00
+++ b/drivers/input/gameport/Makefile	2005-01-03 22:10:04 -08:00
@@ -5,7 +5,7 @@
 # Each configuration option enables a list of files.
 
 obj-$(CONFIG_GAMEPORT)		+= gameport.o
-obj-$(CONFIG_GAMEPORT_CS461X)	+= cs461x.o
+obj-$(CONFIG_GAMEPORT_CS461x)	+= cs461x.o
 obj-$(CONFIG_GAMEPORT_EMU10K1)	+= emu10k1-gp.o
 obj-$(CONFIG_GAMEPORT_FM801)	+= fm801-gp.o
 obj-$(CONFIG_GAMEPORT_L4)	+= lightning.o
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

